### PR TITLE
Post a description of distributed logging for the cb-network system

### PR DESCRIPTION
**Pull request by yunkon-kim**

### CODE REVIEW

File: docs/Distributed-logging-by-ELK-Stack.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+### Logs collected from distributed nodes
+
+Kibana를 통해 분산된 노드에서 수집한 로그를 확인해 보겠습니다. 최근 24시간 이내에 수집되었던 로그 정보를 표시하도록 하였습니다.
+
+첫 번쨰 그림은 ELK Stack의 로그 수집 흐름을 거치면서 부가 정보들이 함께 기록되어 있는 것을 확인 할 수 있습니다.
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/7975459/185302612-deeb6abc-e9c1-453e-9fab-e19e51371a51.png" width="80%" height="80%" >
+</p>
+
+제가 궁금한 것은 멀티클라우드 네트워크 시스템의 각 컴포넌트의 로그만 궁금하기 때문에 message로 필터를 추가하였습니다. 두 번쨰 그림은 그 결과를 나타냅니다.
```
Comment (seokho-son) [inline docs/Distributed-logging-by-ELK-Stack.md:62]:
```suggestion
여기서는 멀티클라우드 네트워크 시스템의 각 컴포넌트의 로그만 확인하면 되는 상황이라, 다른 로그는 보이지 않도록 `message`를 필터로 추가하였습니다. 다음 그림은 그 결과를 나타냅니다.
```

File: docs/Distributed-logging-by-ELK-Stack.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/7975459/185298611-44f1226d-9929-4bcc-933c-740e907c4091.png" width="80%" height="80%" >
+</p>
+
+### Logs collected from distributed nodes
+
+Kibana를 통해 분산된 노드에서 수집한 로그를 확인해 보겠습니다. 최근 24시간 이내에 수집되었던 로그 정보를 표시하도록 하였습니다.
+
+첫 번쨰 그림은 ELK Stack의 로그 수집 흐름을 거치면서 부가 정보들이 함께 기록되어 있는 것을 확인 할 수 있습니다.
```
Comment (seokho-son) [inline docs/Distributed-logging-by-ELK-Stack.md:57]:
```suggestion
첫 번째 그림은 ELK Stack의 로그 수집 흐름을 거치면서 부가 정보들이 함께 기록되어 있는 것을 확인 할 수 있습니다.
```

**Review by seokho-son — COMMENTED** [review]
@yunkon-kim :)

**Comment by dongjae** [comment]
I think Distributed Logging method for CLADNet is very interesting and usefule point. The idea may/can be used in next steps on Cloud-Barista if it verified in near future ^^

File: docs/Distributed-logging-by-ELK-Stack.md (hunk 1 above)
Comment (yunkon-kim) [reply docs/Distributed-logging-by-ELK-Stack.md:62]: @seokho-son 아직 미숙한 :sweat_smile: 문서인데 리뷰해주셨네요 ^^ 감사합니다!!

**Comment by yunkon-kim** [comment]
> I think Distributed Logging method for CLADNet is very interesting and usefule point. The idea may/can be used in next steps on Cloud-Barista if it verified in near future ^^

@dongjae Thanks for sharing your forward-looking view on my simple approach to CLADNet ^^

**Pull request merged**